include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -621,9 +624,148 @@ struct SPhysicalDeviceFeatures
     
     /* Extensions Exposed as Features: */
 
+    /* VK_AMD_buffer_marker */
+    /* VK_AMD_calibrated_timestamps */
+    /* VK_AMD_display_native_hdr */
+    /* VK_AMD_draw_indirect_count */
+    /* VK_AMD_gcn_shader */
+    /* VK_AMD_gpa_interface */
+    /* VK_AMD_gpu_shader_half_float */
+    /* VK_AMD_gpu_shader_half_float_fetch */
+    /* VK_AMD_gpu_shader_int16 */
```

Comment:
deprecatedby="VK_KHR_shader_float16_int8">

Reply:
if it provides no new Vulkan enums, functions, structures or valid usage then can default enable and report as limit